samples/angular-todo/src/app/app.component.ts

```diff
@@ -56,21 +56,21 @@ export class AppComponent implements OnInit {
      * Summary: Configure the SDK
      * ----------------------------------------------------------------------------
      * Details: Below, you will see the following settings:
-     * - clientId: (OAuth2 only) this is the OAuth2 client you created in ForgeRock
-     * - redirectUri: (OAuth2 only) this is the URI/URL of this app too which the
-     *   OAuth flow will redirect
-     * - scope: (OAuth2 only) these are the OAuth scopes that you will request from
+     * - clientId: (OAuth 2.0 only) this is the OAuth 2.0 client you created in ForgeRock, such as `ForgeRockSDKClient`
+     * - redirectUri: (OAuth 2.0 only) this is the URI/URL of this app to which the
+     *   OAuth 2.0 flow redirects
+     * - scope: (OAuth 2.0 only) these are the OAuth scopes that you will request from
      *   ForgeRock
-     * - serverConfig: this includes the baseUrl of your ForgeRock AM, should
-     *   include `/am/` at the end
-     * - realmPath: this is the realm you are wanting to use within ForgeRock
-     * - tree: The authentication journey/tree that you are wanting to use
+     * - serverConfig: this includes the baseUrl of your ForgeRock AM, and should
+     *   include the deployment path at the end, such as `/am/` or `/openam/`
+     * - realmPath: this is the realm to use within ForgeRock. such as `alpha` or `root`
+     * - tree: The authentication journey/tree to use, such as `sdkAuthenticationTree`
      *************************************************************************** */
 
     Config.set({
       clientId: environment.WEB_OAUTH_CLIENT,
       redirectUri: `${window.location.origin}/callback`,
-      scope: 'openid profile email',
+      scope: 'openid profile email address phone',
```

Comment:
Why are we adding the `address` and `phone` scopes here?

Reply:
Purely to match the setup we ask the readers to do when creating the OAuth 2.0 client. I suspect we don't even use them all, but wanted them to (perhaps subconsciously) marry the two steps together.

Reply:
Ok that's fine - just wanted to be sure that the reader will probably have configured the OAuth 2.0 client accordingly. Thanks.

Reply:
What setup are we aligning to here? The [blog article](https://backstage.forgerock.com/docs/sdks/latest/blog/build-protected-angular-web-app.html#public_aka_spa_type_oauth_client_settings) that this sample app corresponds with instructs the reader to only add the first three. If you are referring to the [tutorial from the core docs](https://backstage.forgerock.com/docs/sdks/latest/javascript/tutorial.html), that's the other style of "sample" apps. If we want to align all of the sample apps, we'd have to update quite a few more places in other branches as well.

Reply:
Yeah as you guessed I was aligning to the "Configure the Service" documentation, and hadn't noticed there was a blog post that corresponds to the official samples.
My goal is to unify all server-side setup in a single place, but have it cover all the samples and tutorials on the site. In the meantime, I'll remove the additional scopes from the code. If they use the blog to setup they'll have them, if they use the docs they'll have a few spare scopes in the client, that's all.